WooCommerce/Classes/ViewRelated/InAppFeedback/InAppFeedbackCardViewController.swift

```diff
@@ -12,6 +23,18 @@ final class InAppFeedbackCardViewController: UIViewController {
     /// The stackview containing the `titleLabel` and the horizontal view for the buttons.
     @IBOutlet private var verticalStackView: UIStackView!
 
+    /// SKStoreReviewController type wrapper. Needed for testing
+    private let storeReviewControllerType: SKStoreReviewControllerProtocol.Type
```

Comment:
ah DI'ing a static protocol does look a bit funny 😆

Reply:
Yeah. It looks a bit ugly! 
But I couldn't come up with a better way 😓

Reply:
That's smart and weird at the same time!